File: specification/specs/ingest/simulate_pipeline/SimulatePipelineRequest.ts
@rest_spec_name("ingest.simulate")
class SimulatePipelineRequest extends RequestBase {
  path_parts?: {
    id?: Id;
  }
  query_parameters?: {
    verbose?: boolean;
  }
  body?: {
    docs?: SimulatePipelineDocument[];
    pipeline?: Pipeline;
  }
}

File: output/csharp/Ingest/SimulatePipeline/SimulatePipelineRequest.cs
using Nest.Ingest;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Ingest {

	public class SimulatePipelineRequest : RequestBase {
		
		[DataMember(Name="verbose")]
		public bool Verbose { get; set; }


		[DataMember(Name="docs")]
		public List<SimulatePipelineDocument> Docs { get; set; }


		[DataMember(Name="pipeline")]
		public Pipeline Pipeline { get; set; }

	}
}
